src/test/java/javatests/JsonConverterTest.java

```diff
@@ -126,4 +97,36 @@ public void testGetConfigMethodWithFakeVendorID() {
     assertTrue("getConfig() didn't return null when it should have.",
       expectedResponse == actualResponse);
   }
+
+  /** Test that writeFile() returns a File with the expected content. */
+  @Test
+  public void testWriteFile() {
+    JsonConverter converter = new JsonConverter();
+    vendor.addAccount(account);
+
+    String expectedResponse = String.format("{\"legacy_vendor_id\":%s," +
+      "\"next_gen_vendor_id\":%d,\"accounts\":[{\"legacy_account_id\":%s," +
+      "\"next_gen_customer_id\":%d,\"settlement_attributes\":{" +
+      "\"currency_code\":%s,\"direction\":%s,\"entity\":%s}," +
+      "\"settlement_config\":{\"matching_mode\":%s},\"account_id\":%s," +
+      "\"aggregation_mode\":%s}}]}", LEGACY_VENDOR_ID, NEXT_GEN_VENDOR_ID,
+      LEGACY_ACCOUNT_ID, NEXT_GEN_ACCOUNT_ID, CURRENCY, DIRECTION, ENTITY, 
+      MATCHING_MODE, ACCOUNT_ID,AGGREGATION_MODE);
+    String actualResponse = "";
+
+    File file = converter.writeFile(vendor.getVendorID(), vendor.buildJsonConfig());
+    Scanner input = null;
+    try {
+      input = new Scanner(file);
+    } catch(FileNotFoundException e) {
+      e.printStackTrace();
```

Comment:
Is this meant to fail the test? It is okay to remove the try/catch so this throws. That will fail the test and print the error to console.

Reply:
Gotcha